src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const totalPay = days * rentPrice;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const threeDays = 3;
+  const sevenDays = 7;
```

Comment:
What if the number of days will change ?

Reply:
done